### DISCUSSION TITLE
Copy/paste from one Budibase app to another

### DISCUSSION THREAD

**Discussion by Pachat** [Budibase/budibase#2875] [Ideas]
Is it possible to merge budibase apps developed independently in into one budibase app?

**Comment by mjashanks** [comment]
Not at the moment. I have often found myself wanting to copy the items below between apps...
1. Screens
2. Components within screens
3. Datasources

is this what you are needing too? Do you have anything to add to that?

**Comment by Pachat** [comment]
Yes. I may add: menus, js and css resources including images, if need be.

**Comment by edrisi6** [comment]
any news on this? It will make budibase a power engine like no other. it will save time and will make production much sweeter.

**Comment by NicolasFabre81** [comment]
Hello @mjashanks , any news on this ?

**Reply by Kakovisk** [reply]
Every time I create a new application, I would like to bring components or pages with me. Unfortunately, I have to redo everything. Any ideas? How is this topic going?

**Reply by bjornclauw** [reply]
+1

**Comment by MarceloArraes** [comment]
+1
